app/scripts/controllers/metametrics-data-deletion/metametrics-data-deletion.ts

```diff
@@ -0,0 +1,125 @@
+import { ObservableStore } from '@metamask/obs-store';
+import MetaMetricsController from '../metametrics';
+import {
+  createDataDeletionRegulationTask,
+  fetchDeletionRegulationStatus,
+} from './services/services';
+
+export type DataDeleteDate = string;
+export type DataDeleteRegulationId = string;
+
+export type RegulationId = Record<string, string>;
+export type CurrentRegulationStatus = Record<string, Record<string, string>>;
+export type DeleteRegulationAPIResponse = {
```

Comment:
We're creating an import cycle here because this type is used by `services.ts`, but we're importing `services` here. We could eliminate this cycle by moving these types to a separate file, e.g. `types.ts`